deployment/hasura/migrations/AerieMerlin/22_constraint_violation_schema/up.sql

```diff
@@ -0,0 +1,6 @@
+-- The schema of the database is unchanged, but the JSON schema of the constraint violations in constraint_run is different.
+-- Theoretically we could use SQL to transform the schemas of existing JSON objects, but I'd rather just delete them.
+-- programming is hard, ok
+delete from constraint_run;
```

Comment:
Minor: since `constraint_run.violations` can now contain an object without any violations it ought to be renamed.

Reply:
done